.golangci.yaml

```diff
@@ -55,7 +52,6 @@ linters-settings:
 
 
 linters:
```

Comment:
If we're not disabling default, and only enabling what's already enabled by default, could we remove this whole section?

Reply:
possibly we could, but with that we can 
1) see which linters are enabled without looking to linter documentation
2) if we upgrade the linter and the default linters change, we will not be affected

Reply:
1 is not true already, with current setup we have:
- [errcheck](https://golangci-lint.run/usage/linters/#errcheck)
- [gosimple](https://golangci-lint.run/usage/linters/#gosimple)
- [govet](https://golangci-lint.run/usage/linters/#govet)
- [ineffassign](https://golangci-lint.run/usage/linters/#ineffassign)
- [staticcheck](https://golangci-lint.run/usage/linters/#staticcheck)
- [unused](https://golangci-lint.run/usage/linters/#unused)

and 2 is also not true, if they add a default linter, we will have it enabled.

This would be true if we keep `disable-all: true`

Reply:
with `disable-all: true` all is well, thank you 👌